.github/gnav.md

```diff
@@ -0,0 +1,30 @@
+## Description
+
+## Related Issue
+Resolves: [MWPW-111111](https://jira.corp.adobe.com/browse/MWPW-111111)
+
+## Testing instructions
+
+## Screenshots (if appropriate):
+
+## Types of changes
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
```

Comment:
Not 100% sure we need these, as we could simply use labels 🤷

Reply:
Fine by me to remove it!